tesseract_kinematics/kdl/include/tesseract_kinematics/kdl/kdl_utils.h

```diff
@@ -102,6 +102,22 @@ struct KDLChainData
   std::vector<std::pair<std::string, std::string>> chains; /**< The chains used to create the object */
 };
 
+/**
+ * @brief The KDLConfig struct
+ *
+ * This contains parameters that can be used to customize the KDL solver for your application.
+ * They are ultimately passed to the constuctor of the undelying ChainIkSolver.
+ *
+ * The defaults provided here are the same defaults imposed by the KDL library.
+ */
+struct KDLConfig
+{
+  Eigen::VectorXd weights = Eigen::VectorXd::Constant(6, 1.0);
+  double eps = 1E-5;
+  int max_iterations = 500;
+  double eps_joints = 1E-15;
```

Comment:
nitpik, please use `{}`
``` c++
  Eigen::VectorXd weights {Eigen::VectorXd::Constant(6, 1.0)};
  double eps {1E-5};
  int max_iterations {500};
  double eps_joints {1E-15};
```

Reply:
Also are these default the current ones used by KDL?

Reply:
Yes, [per here](https://docs.ros.org/en/kinetic/api/orocos_kdl/html/classKDL_1_1ChainIkSolverPos__LMA.html#a5322326848d618e0ec2af8683344274e)